packs/data/spell-effects.db/spell-effect-blessing-of-defiance.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Blessing of Defiance",
     "system": {
         "description": {
-            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Blessing of Defiance]{Blessing of Defiance}</em></p>\n<p>Choose one saving throw: Fortitude, Reflex, or Will. The target gets a +4 status bonus to that saving throw for 1 round. If you choose Fortitude, the target gains resistance 15 to poison. If you choose Reflex, the target gains a +15-foot status bonus to its Speeds. If you choose Will, the target gains resistance 15 to mental damage.</p>"
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Blessing of Defiance]{Blessing of Defiance}</em></p>\n<p>Choose one saving throw: Fortitude, Reflex, or Will. The target gets a +4 status bonus to that saving throw for 1 round.</p>"
```

Comment:
Instead of just deleting the text let's make it actually work properly so the text is right. We'll need a second ChoiceSet for the 1 action / 2 action version and then 3 REs predicated on 2 action + the choice of save.

Reply:
The 1 action variant doesn’t grant those benefits. 
https://2e.aonprd.com/Spells.aspx?ID=871

Reply:
Oh, I see. Reduce to a single effect and migrate.